frontend/lib/resolution/Resolver.cpp

```diff
@@ -3032,6 +3033,10 @@ shouldSkipCallResolution(Resolver* rv, const uast::AstNode* callLike,
     ci.numActuals() == 1 &&
     (ci.actual(0).type().isType() || ci.actual(0).type().isParam());
 
+  allowGenericTypes |=
```

Comment:
Can you add a comment indicating that this is a workaround, and open a private issue?